client/src/App.tsx

```diff
@@ -1,35 +1,15 @@
-import { useState } from 'react';
-import reactLogo from './assets/react.svg';
-import viteLogo from '../../../../vite.svg';
-import './App.css';
-
-function App() {
-  const [count, setCount] = useState(0);
+import { Routes, Route } from 'react-router-dom';
+import { Header } from './components/Header';
+import { Entries } from './pages/Entries';
+import { EntryForm } from './pages/EntryForm';
 
+export default function App() {
   return (
-    <>
-      <div>
-        <a href="https://vite.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+    <Routes>
+      <Route path="/" element={<Header />}>
+        <Route index element={<Entries />} />
+        <Route path="entry/:entryId" element={<EntryForm />} />
+      </Route>
```

Comment:
It would be good to add a "not found" path. That will protect you if the user tries to hack your site by typing into the URL.